src/molgenis/capice/cli/args_handler_predict.py

```diff
@@ -98,6 +100,24 @@ def _validate_model_attributes(self, model):
                 self.parser.error(f'Unable to locate attribute {attribute} in model file!')
 
     def _validate_model_version(self, model):
-        if not model.CAPICE_version == __version__:
-            self.parser.error(f'Model version {model.CAPICE_version} '
-                              f'does not match CAPICE version: {__version__}!')
+        model_version = match(Versioning.VALIDATION_REGEX.value, model.CAPICE_version)
+        if model_version is None:
+            self.parser.error(f'Model version does not adhere to correct format: '
+                              f'{model.CAPICE_version}!')
+
+        capice_version = match(Versioning.VALIDATION_REGEX.value, __version__)
+        if model_version.group('major') != capice_version.group('major'):
+            self.parser.error(f'Model major version {model.CAPICE_version} does not match '
+                              f'with CAPICE: {__version__}!')
+
+        if model_version.group('prerelease') is not None \
+                or capice_version.group('prerelease') is not None:
```

Comment:
How would `if model_version.group('prerelease') or capice_version.group('prerelease'):` perform? It might fit the 100 maximum line length then too perhaps?